### ISSUE TITLE
OptInDialogTest time outs on Kokoro

### ISSUE THREAD

**Issue by chanseokoh** [GoogleCloudPlatform/google-cloud-eclipse#2639]
I've feared of this possibility. My recent changes to the test to simulate dialog closing doesn't seem to work on Windows.

```
Running com.google.cloud.tools.eclipse.usagetracker.OptInDialogTest
[ERROR] Timeout 600 s exceeded. Process was killed.
```

**Comment by briandealwis** [comment]
Do you know the actual cause here?  There's no issue around calling `Shell#close()` on Windows.  I wonder if we're seeing an issue as the `OptInDialog` is being built but not actually opened?

**Comment by chanseokoh** [comment]
It wasn't about `close()` not working, of course.

I don't know exactly, but my guess for the reason is that I was attaching a focus listener to the root shell and not to the dialog shell, and the moment the root shell gains the focus may not always be after the dialog shell opens. The focus listener trick was kind of a hack, from my perspective. In fact, DISPOSE_SHELL wasn't working previously on Linux. (Do you remember my comment about that?) Now everything works robustly, and I think this is the correct way.

**Comment by chanseokoh** [comment]
Will be fixed by #2642, if you haven't noticed.

**Comment by briandealwis** [comment]
Sorry, I must have seen these out-of-sequence as I thought the new method had failed.
